metrics/internal/collectors/registry.go

```diff
@@ -27,15 +27,18 @@ func RegisterCustomResourceCollectors(registry *prometheus.Registry, opts *optio
 	cephClusterCollector := NewCephClusterCollector(opts)
 	OBMetricsCollector := NewObjectBucketCollector(opts)
 	clusterAdvanceFeatureCollector := NewClusterAdvancedFeatureCollector(opts)
+	storageConsumerCollector := NewStorageConsumerCollector(opts)
```

Comment:
You may require a similar `nil` check like the below `clusterAdvanceFeatureCollector` object, as there is a `nil` being returned from `NewStorageConsumerCollector()` function (if you add the changes suggested) =)

Reply:
Done

Reply:
Done